plasmapy/dispersion/numerical/tests/test_stix_.py

```diff
@@ -0,0 +1,97 @@
+"Test functinality of Stix in plasmapy.dispersion.numerical.stix_"
+from astropy import units as u
+import numpy as np
+import pytest
+
+from plasmapy.dispersion.numerical.stix_ import stix
+
+class TestStix:
```

Comment:
```suggestion
from plasmapy.dispersion.numerical.stix_ import stix


class TestStix:
```

PEP8 standard: There should always be two blank lines before the definition of a class.